### ISSUE TITLE
DS4Windows doesn't start after selecting profiles destination

### ISSUE THREAD

**Issue by GeraltV47** [Ryochan7/DS4Windows#2338]
**Describe the bug**
DS4Windows doesn't start after selecting profiles destination.

**To Reproduce**
1. Go to 'DS4Windows main folder'
2. Click on 'DS4Windows'

**Expected behavior**
Just application to run.

**Screenshots and Logs**
If applicable, add screenshots to help explain your problem. Also, please add the most
log file (ds4windows_log.txt) from the Logs folder in your DS4Windows config folder.

**Desktop (please complete the following information):**
 - Controller Make and Model: Sony DS4 v.2 (CUH-ZCT2E)
 - OS: [e.g. Windows 10 Pro Build 2004]
 - DS4Windows Version [e.g. 3.0.13]

**Additional context**

**Comment by GeraltV47** [comment]
The other thing worth to mention is that i just reinstalled the windows while ago, may it be caused by missing libaries?

**Comment by mika-n** [comment]
#1196 
Please provide the latest logfile.

And do you mean that DS4Windows app keeps asking the profile folder each time you try to launch the app? Or it asked this only once after installation, but after that the app doesn't seem to open?

If the app keeps asking profile location then it means (in 99.999999% cases) that you have some file permission problems (ie. ds4windows app fails to create required config file, so this is why it keeps asking the profile location over and over again). You should figure out why your profile folder doesn't allow DS4Windows app to create config files. Maybe some over aggressive virus scanner blocking the app?

**Comment by GeraltV47** [comment]
It asked me just once, where i can find logfile?

**Comment by mika-n** [comment]
read the linked and pinned "Read this before posting a new issue ticket/discussion post" thread. Logfile details are described there

**Comment by GeraltV47** [comment]
[ds4windows_log_20210721.0.txt](https://github.com/Ryochan7/DS4Windows/files/6857610/ds4windows_log_20210721.0.txt)

**Comment by mika-n** [comment]
`2021-07-21 09:22:14.7524|ERROR|System.PlatformNotSupportedException: The native library 'C:\Windows\Microsoft.NET\Framework64\v4.0.30319\wminet_utils.dll' does not have all required functions. Please, update the .NET Framework.
`

Looks like you have an old .NET runtime libraries. See the link list.
https://github.com/Ryochan7/DS4Windows/wiki/Troubleshooting#link-list

Except that I don't understand why it is looking for .NET Framework 4.x libraries because DS4Windows app uses .NET 5.x libraries nowadays. Weird. Anyway, install the latest version from both .NET Framework 4.8.x x64 and .NET 5.x for desktop apps libraries.
https://dotnet.microsoft.com/download/dotnet/5.0/runtime
https://dotnet.microsoft.com/download/dotnet-framework

**Comment by GeraltV47** [comment]
When i try to install NET 4.8 it says at start that i cant install .NET Framework 4,8 on that version of Windows.

**Comment by GeraltV47** [comment]
OK, Windows installed some updates and DS4W started, can you tell me what it could install to avoid such a problem in furure? I plan to disable automatical windows updates because they eat all my mobile data, im connected to internet through my phone tethering.

**Comment by GeraltV47** [comment]
I found another isssue, i posted it before but i was on windows 7 back then that required installing a previous version of vigembus driver, it just finished installing and when i start DS4W and see step by step window, it tells that installation of vigembus failed, and button "install 360 driver" is greyed out.

**Comment by mika-n** [comment]
make sure you downloaded and installed 64bit X64 versions of DS4Windows app, .NET Framwork 4.8, .NET 5.x FOR DESKTOP apps and ViGemBus driver. Forget all those x86 32bit app versions (the installation filename should have x64 tag and not x86).

And if you don't install all Windows updates then you are on your own because we really cannot keep up with support on old and potentially buggy/limited versions of Windows components.

**Comment by GeraltV47** [comment]
ok thanks
